pkg/repo/goget_test.go

```diff
@@ -1,16 +1,17 @@
 package repo
 
 import (
-	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/spf13/afero"
 )
 
 func Test_Download(t *testing.T) {
 	version := "v0.1.8"
 	gitURI := "github.com/bketelsen/captainhook"
-
-	fetcher, err := NewGenericFetcher(gitURI, version)
+	fs := afero.NewMemMapFs()
```

Comment:
try running with osFs, not sure how this will cooperate with vgo

Reply:
Sure... vgo knows nothing about afero.thanks!